src/js/common/polyfills.js

```diff
@@ -24,3 +24,13 @@ if (!Modernizr.fetch) {
 
 // This polyfill has its own test logic so no need to conditionally require.
 require('polyfill-function-prototype-bind');
+
+// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
+// NOTE: This also just stops the user from scrolling when hovering over a number
+//  input that has focus. This isn't ideal, but until a solution is found to that,
+//  this is what we've got.
+document.addEventListener('mousewheel', (event) => {
```

Comment:
@cehsu Yeah, I wasn't sure about that. I was shying away from the accessibility implications there might be if we use `.blur()`, but I'll let @goldenmeanie weigh in on what's less bad.